docs/tools/cli/billing-group.rst

```diff
@@ -0,0 +1,243 @@
+  * - ``--billing-extra-text``
+    - Information to include with an invoice such as a cost center number
+  * - ``--billing-email``
+    - Email for the billing contact
+  * - ``--company``
+    - Company name
+  * - ``--address-line``
+    - First line of address
+  * - ``--country-code``
+    - `Code <https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements>`_ for the billing country
+  * - ``--city``
+    - City
+  * - ``--state``
+    - State / Province
+  * - ``--zip-code``
+    - ZIP / Post Code
+
+**Example:** Create a billing-group named ``qa-dept``.
+
+::
+
+  avn billing-group create qa-dept --account-t c59dde4j9 --billing-currency EUR --billing-email billing@testers.dev --company testers --address-line "1 No Way" --country-code SE --city Stockholm
+
+**Example:** Rename a billing-group 
+
+::
+
+  avn billing-group update \
+    55b0e547-58f9-48de-8808-807d385d1f95 \
+    --name qa-department 
+
+``avn billing-group credits-claim``
+''''''''''''''''''''''''''''''''''''
+
+Claim a credit code within your biling-group
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Parameter
+    - Information
+  * - ``id``
+    - The ID of your billing group
+  * - ``code``
+    - Credit Code
+
+**Example:** Claim a credit code in your billing-group
+
+::
+
+  avn billing-group credits-claim 55b0e547-58f9-48de-8808-807d385d1f95 sneaky-crab
+
+``avn billing-group credits-list``
+''''''''''''''''''''''''''''''''''''
+
+Lists all the credits redeemed in your billing-group
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Parameter
+    - Information
+  * - ``id``
+    - The ID of your billing group
+
+**Example:** List credits claimed in your billing-group
+
+::
+
+  avn billing-group credits-list 55b0e547-58f9-48de-8808-807d385d1f95
+
+``avn billing-group delete``
+''''''''''''''''''''''''''''''''''''
+
+Lists all the credits redeemed in your billing-group
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Parameter
+    - Information
+  * - ``id``
+    - The ID of your billing group
+
+**Example:** Delete your billing-group
+
+::
+
+  avn billing-group delete 55b0e547-58f9-48de-8808-807d385d1f95
+
+``avn billing-group events``
+'''''''''''''''''''''''''''''
+
+List the events that have occurred for a given billing-group 
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Parameter
+    - Information
+  * - ``id``
+    - The ID of your billing group
+
+**Example:** List events in your billing-group
+
+::
+
+  avn billing-group events 55b0e547-58f9-48de-8808-807d385d1f95
+
+``avn billing-group get``
+''''''''''''''''''''''''''
+
+Get details for a given billing-group
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Parameter
+    - Information
+  * - ``id``
+    - The ID of your billing group
+
+**Example:** Get details for your billing-group
+
+::
+
+  avn billing-group get 55b0e547-58f9-48de-8808-807d385d1f95
```

Comment:
Is it possible to have a sample output? of the get command?

Reply:
This was laziness on my part because I did not want to sanitise output. Will run this in a dev environment, yep